### DISCUSSION TITLE
lsp go to definition is not working for scss modules in react app

### DISCUSSION THREAD

**Discussion by victororlyk** [helix-editor/helix#10370] [Troubleshooting]
### Summary

when using  scss modules and `gd` it will lead to some wrong file ` node_modules/react-scripts/lib/react-app.d.ts` isntead of scss module file which leaves in the same folder. 

### Reproduction Steps

 

I tried this:

1. npx create-react-app my-app
2. cd my-app
3. touch src/some.module.scss create some classes in there
4. import that some.module.scss in some component and try using `gd` on styles.class

I expected this to happen:
i expect to go that class 
Instead, this happened:
it throws me in node_modules/react-scripts/lib/react-app.d.ts

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

alacritty

### Helix Version

helix 22.12 (96ff64a8)

**Comment by kostaskriaridis** [comment]
Have the same issue

**Comment by askreet** [comment]
Part of the bug template is the Helix log capture. For LSP issues it's very likely there's useful stuff in there. Can you take a look?

**Comment by dead10ck** [comment]
The definition that helix jumps to comes from the LS, not helix itself. Does this happen in other editors with the same LS?
